uploader.json

```diff
@@ -736,6 +736,11 @@
             "pattern": [],
             "symbolic_name": "rpm__V_packages"
         },
+        {
+            "command": "/usr/sbin/gluster volume info",
+            "pattern": [],
+            "symbolic_name": "gluster_v_info"
+        },
```

Comment:
please move the gluster_v_info defs between these two defs. We need to try to keep this in alphabetical order.
```
        {
            "command": "/usr/sbin/getsebool -a",
            "pattern": [],
            "symbolic_name": "getsebool"
        },
        {
            "command": "/bin/ls -l /boot/grub/grub.conf",
            "pattern": [],
            "symbolic_name": "grub1_config_perms"
        },
```